pkg/frontend/authenticate.go

```diff
@@ -2476,7 +2476,7 @@ func doAlterUser(ctx context.Context, ses *Session, au *tree.AlterUser) error {
 			err = moerr.NewInternalError(ctx, "Operation ALTER USER failed for '%s'@'%s', user does't exist", user.Username, user.Hostname)
 			goto handleFailed
 		} else {
-			return err
+			goto handleSucceeded
```

Comment:
can this goto be replaced by defer?

Reply:
> can this goto be replaced by defer?

i try to fix it recently. but not  in this pr.